core/src/banking_stage.rs

```diff
@@ -816,10 +879,13 @@ impl BankingStage {
         unprocessed_packets: &mut UnprocessedPackets,
         packets: Packets,
         packet_indexes: Vec<usize>,
+        dropped_batches_count: &mut usize,
+        batch_limit: usize,
     ) {
         if !packet_indexes.is_empty() {
-            if unprocessed_packets.len() > 400 {
+            if unprocessed_packets.len() > batch_limit {
```

Comment:
This will let the unprocessed_packets len to go 1 higher than batch_limit (as we are pushing the new batch at line 888). Not a big deal, just that the variable name is not aligning with the code..

The check can be changed to `>=`